.gitmodules

```diff
@@ -1,19 +1,22 @@
 [submodule "asn1_codec"]
 	path = asn1_codec
-	url = https://github.com/usdot-jpo-ode/asn1_codec.git
+	url = https://github.com/CDOT-CV/asn1_codec.git
+	branch = confluent-cloud-integration
```

Comment:
Should we still be referencing the individual CC integration branches for the submodules, or should we target our dev branches?

Reply:
Just wanted to follow up on this @drewjj

Reply:
Changing the submodule references will get complicated fast when we try to create PRs back to the USDOT repo. Useful for testing these branches, but probably don't want to commit to our dev branch

Reply:
These changes have been reverted and this file references usdot's repositories once more.